rest-openehr/src/main/java/org/ehrbase/rest/openehr/experimental/ItemTagController.java

```diff
@@ -0,0 +1,277 @@
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.lang.Nullable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Experimental REST interface for {@link <a href="https://specifications.openehr.org/releases/RM/latest/ehr.html#tags">4.2.6. Tags</a>}
+ */
+@ConditionalOnMissingBean(name = "primaryopenehritemtagcontroller")
+@RestController
+@RequestMapping(
+        path = "${ehrbase.rest.experimental.tags.context-path:/rest/experimental/tags}/ehr",
+        produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+@ConditionalOnProperty(name = "ehrbase.rest.experimental.tags.enabled", havingValue = "true")
+public class ItemTagController extends BaseController implements ItemTagApiSpecification {
+
+    private final ItemTagService itemTagService;
+
+    public ItemTagController(ItemTagService itemTagService) {
+        this.itemTagService = itemTagService;
+    }
+
+    // --- EHR_STATUS ---
+
+    @PutMapping(
+            value = "/{ehr_id}/ehr_status/{versioned_object_uid}/item_tag",
+            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+    @ResponseStatus(value = HttpStatus.OK)
+    public ResponseEntity<Object> upsertEhrStatusItemTags(
+            @RequestHeader(value = OPENEHR_VERSION, required = false) String openehrVersion,
+            @RequestHeader(value = OPENEHR_AUDIT_DETAILS, required = false) String openehrAuditDetails,
+            @RequestHeader(value = HttpHeaders.CONTENT_TYPE) String contentType,
+            @RequestHeader(value = HttpHeaders.ACCEPT, required = false) String accept,
+            @RequestHeader(value = PREFER, required = false) String prefer,
+            @PathVariable(value = "ehr_id") String ehrIdString,
+            @PathVariable(value = "versioned_object_uid") String versionedObjectUid,
+            @RequestBody Collection<ItemTagDto> itemTags) {
+
+        return upsertItemTags(
+                accept,
+                prefer,
+                ehrIdString,
+                versionedObjectUid,
```

Comment:
Setting a different `target` than the `versionedObjectUid` through the DTO should maybe result in a 400 Bad Request. Same for the ownerId (Also applies to the composition endpoint)